storm-core/src/jvm/backtype/storm/utils/TransferDrainer.java

```diff
@@ -26,10 +26,10 @@
 
 public class TransferDrainer {
 
-  private HashMap<String, ArrayList<ArrayList<TaskMessage>>> bundles = new HashMap();
+  private HashMap<Integer, ArrayList<ArrayList<TaskMessage>>> bundles = new HashMap();
   
-  public void add(HashMap<String, ArrayList<TaskMessage>> workerTupleSetMap) {
-    for (String key : workerTupleSetMap.keySet()) {
+  public void add(HashMap<Integer, ArrayList<TaskMessage>> taskTupleSetMap) {
```

Comment:
@d2r 
Yes, we don't need this since we're adding tuples and sending it immediately. 
I'll make it private and add parameter to send. Thanks!